app/src/main/res/layout/tutorial.xml

```diff
@@ -16,21 +15,21 @@
         android:text="Enter Your Phone Number : "
         android:textColor="#000000"
         android:textSize="20dp" />
-
     <EditText
         android:id="@+id/phonenumberEditText"
         android:layout_width="300dp"
         android:layout_height="wrap_content"
-        android:layout_marginStart="20dp"
         android:layout_marginLeft="20dp"
-        android:hint="010-1234-5678"
-        android:inputType="phone"/>
+        android:hint="폰번호를 입력하세요."
+        android:inputType="textPersonName"
+        android:text="Enter Your Phone number"
```

Comment:
When there is no saved user data, the text value must be empty because the hint must be output.
And why did you specify inputType as `textPersonName`? 
This is EditText receiving user's phone number, so you need to change the inputType again.

```suggestion
        android:hint="Enter Your Phone number"
        android:inputType="phone"
```